lib/find-author-name.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const parse = require('parse-github-url');
+
+module.exports = ({repoURL, dirname}) => {
+	if (repoURL) {
+		return parse(repoURL).owner;
+	}
+
+	const pkgJsonFile = path.resolve(dirname, 'package.json');
+	if (fs.existsSync(pkgJsonFile)) {
```

Comment:
`existsSync` is discouraged for exactly this reason. You should never use it before reading from disk. Instead, try requiring and just try/catch and ignore.